webapp/src/pages/eventConfig/components/EventConfigComponent.js

```diff
@@ -302,21 +306,22 @@ class EventConfigComponent extends Component {
       "Call": ALL_DATE_FIELDS.slice(0, 3)
     }
     const requiredDateFields = requiredDateFields_by_event[event_type];
-    const future_date = new Date();
-    future_date.setFullYear(2099);
+
```

Comment:
This is awesome! Some suggestions to improve the readability a bit: firstly it makes sense as you suggested in your comment to move this const to where ALL_DATE_FIELDS is defined. Secondly the `ALL_DATE_FIELDS.slice(0, 4)` while it works well, is not as readable as being more explicit about it. I'd consider adding more constants for your date pairs and then do something like this, which is less error prone (in fact it looks like you forgot to add the start and end dates in your code?) 

```
const ALL_DATE_FIELDS = [APPLICATION_DATES, REVIEW_DATES, SELECTION_DATES, OFFER_DATES, REGISTRATION_DATES, EVENT_DATES];

const REQUIRED_DATES_BY_EVENT_TYPE = {
      "Event": ALL_DATE_FIELDS,
      "Programme": ALL_DATE_FIELDS,
      "Award": [APPLICATION_DATES, REVIEW_DATES, SELECTION_DATES, OFFER_DATES, EVENT_DATES],
      "Call": [APPLICATION_DATES, REVIEW_DATES, SELECTION_DATES, EVENT_DATES]
    }
```